opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -507,7 +518,7 @@ def __init__(
     ):
         # TODO: How should multiple TracerSources behave? Should they get their own contexts?
         # This could be done by adding `str(id(self))` to the slot name.
-        self._current_span_slot = Context.register_slot("current_span")
+        self._current_span_name = "current_span"
```

Comment:
I think there is something wrong, are there two slots with an almost equal name?, `current-span` and `current_span`?